package.json

```diff
@@ -139,7 +139,7 @@
     "joi-browser": "^13.4.0",
     "lint-staged": "^7.2.2",
     "prettier": "^1.8.2",
-    "pubsweet": "^2.3.0",
+    "pubsweet": "^2.2.8",
```

Comment:
Don't think we should be downgrading these - probably what is causing the failures in the pipeline.